src/main/java/com/patchworkmc/jar/ForgeModJar.java

```diff
@@ -0,0 +1,54 @@
+package com.patchworkmc.jar;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import com.patchworkmc.manifest.accesstransformer.AccessTransformerList;
+import com.patchworkmc.manifest.converter.com.patchworkmc.manifest.AccessTransformerListMerger;
+import com.patchworkmc.manifest.mod.ModManifest;
+import com.patchworkmc.manifest.mod.ModManifestEntry;
+
+public class ForgeModJar {
+	private Path jarPath;
+	private ModManifest manifest;
+	private AccessTransformerList accessTransformers;
+
+	public ForgeModJar(Path jarPath, ModManifest manifest, AccessTransformerList list) {
+		this.jarPath = jarPath;
+		this.manifest = manifest;
+		this.accessTransformers = list;
+	}
+
+	public void addDependencyJars(List<ForgeModJar> modJars) {
+		for (ForgeModJar mod : modJars) {
+			if (mod.equals(this)) {
+				continue;
+			}
+
+			boolean depends = false;
+
+			for (ModManifestEntry modManifestEntry : getManifest().getMods()) {
```

Comment:
I think it would make more sense to compile a `Set` outside of the loop with the dependency mod IDs and then test against that set in the loop, instead of iterating every time.

Reply:
that was actually a logic mistake, it's supposed to iterate through every modid in the proposed dependency, not the object's own manifest